cmd/node-cache/app/configmap.go

```diff
@@ -16,12 +18,19 @@ const (
     errors
     cache {{.CacheTTL}}
     bind {{.LocalIP}}
-    forward . {{.UpstreamServers}} {
-      force_tcp
-    }
+    forward . {{.UpstreamServers}}
 }
 `  // cache TTL is 30s by default
-	defaultTTL = 30
+	defaultTTL    = 30
+	upstreamBlock = `
+    forward . __PILLAR__UPSTREAM__SERVERS__ {
+            force_tcp
```

Comment:
This is to match the indentation in the coreFile, which used 8 spaces within the forward block instead of 4 - https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml#L67